onnxruntime/contrib_ops/cpu/bert/attention.cc

```diff
@@ -33,13 +33,13 @@ AttentionBase::AttentionBase(const OpKernelInfo& info) {
 Status AttentionBase::CheckInputs(const TensorShape& input_shape,
                                   const TensorShape& weights_shape,
                                   const TensorShape& bias_shape,
-                                  const Tensor* mask_index,
+                                  const Tensor*& mask_index,
```

Comment:
is it that we want to keep the input type of this function as all const?

Reply:
We will update mask_index to nullptr in the function when the shape is like (1, 1) or (batch_size, 1)